src/main/java/edu/kis/powp/jobs2d/command/history/HistoryEntry.java

```diff
@@ -0,0 +1,29 @@
+package edu.kis.powp.jobs2d.command.history;
+
+import edu.kis.powp.jobs2d.command.DriverCommand;
+
+public class HistoryEntry {
+
+    private String date;
+    private String commandName;
+    private DriverCommand command;
+
+    public String getCommandName() {
+        return commandName;
+    }
+
+    public HistoryEntry(String date, String commandName, DriverCommand command) {
+        this.date = date;
+        this.commandName = commandName;
+        this.command = command;
```

Comment:
Can you ensure that the command passed by the parameter won't be changed by others?

Reply:
Yes, in DriverCommandManager in method setCurrentCommand we are creating new ImmutableComplexCommand which later is going to be present in history.

Reply:
ok